lib/action_policy/testing.rb

```diff
@@ -25,18 +25,20 @@ def inspect
       end
 
       class Scoping # :nodoc:
-        attr_reader :policy, :type, :name
+        attr_reader :policy, :type, :name, :scope_options
 
-        def initialize(policy, type, name)
+        def initialize(policy, type, name, scope_options)
           @policy = policy
           @type = type
           @name = name
+          @scope_options = scope_options
         end
 
-        def matches?(policy_class, actual_type, actual_name)
+        def matches?(policy_class, actual_type, actual_name, actual_scope_options)
           policy_class == policy.class &&
             type == actual_type &&
-            name == actual_name
+            name == actual_name &&
+            scope_options == actual_scope_options
         end
 
         def inspect
```

Comment:
Let's add `scope_options` to `#inspect` too.

NOTE: if we add support for composed matchers we should handle them differently to have human-readable output (see, for example, https://github.com/palkan/action-cable-testing/blob/master/lib/rspec/rails/matchers/action_cable.rb#L151).